.github/workflows/create_documentation.yml

```diff
@@ -55,6 +55,7 @@ jobs:
         run: |
           cd build
           make doc
+          cp -r coverageReport doc/pages/coverageReport
```

Comment:
If we copy the coverage report into the documentation: Do we really need a separate upload of the coverage report (line 53 and 66)

Reply:
I guess it can be usefull in the case we only want to check the new coverage report for a PR. But I guess in this case a seperate action would be more helpfull.